### PR TITLE
Update SDK to 4.26.0

### PR DESCRIPTION
**Pull request by wangjiejacques**

### CODE REVIEW

File: SPM Sample/SPM-sample-app.xcodeproj/project.pbxproj [hunk 1]
```diff
@@ -525,8 +525,8 @@
 			isa = XCRemoteSwiftPackageReference;
 			repositoryURL = "https://github.com/ContentSquare/CS_iOS_SDK.git";
 			requirement = {
-				kind = exactVersion;
-				version = 4.25.2;
+				kind = upToNextMajorVersion;
+				minimumVersion = 4.25.2;
```
Comment (christianbilodeau) [inline SPM Sample/SPM-sample-app.xcodeproj/project.pbxproj:529]: I personally prefer having "exact" for two reasons:

- it leaves some git history that the version was updated and therefore the sample app tested
- it removes the need to discuss it every time we change it

File: SPM Sample/SPM-sample-app.xcodeproj/xcshareddata/xcschemes/SPM-sample-app.xcscheme [hunk 2]
```diff
@@ -0,0 +1,77 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (christianbilodeau) [inline SPM Sample/SPM-sample-app.xcodeproj/xcshareddata/xcschemes/SPM-sample-app.xcscheme:1]: Why is this file added now? 🤔

Comment (wangjiejacques) [reply SPM Sample/SPM-sample-app.xcodeproj/xcshareddata/xcschemes/SPM-sample-app.xcscheme:1]: I don't know, maybe added by my xcode 15

**Review by christianbilodeau — APPROVED** [review]
